grants/speculative/PolkaHub.md

```diff
@@ -0,0 +1,235 @@
+# PolkaHub. Scalable Blockchain Infrastructure
+
+
```

Comment:
## Updates from previous version

We will build using Kubernetes instead of Flynn. This will somewhat increase the scope of our work (which we will absorb at our expense), however as a trade-off it will allow us to maintain a  more unified development process of PaaS by coding it entirely on Rust (Kubernetes) vs switching to Go (Flynn).